src/customHooks/useDocumentTitle.js

```diff
@@ -8,6 +8,7 @@ const useDocumentTitle = () => {
 
   useEffect(() => {
     window.document.title = `FrontEnd|${title_}`;
+    window.scrollTo({ top: 0, behavior: "auto" });
   }, [title_]);
```

Comment:
you can use `title` name simply if there are more words like `titlePrice or titleXYZ` thn you can use `title_XYZ`
what u have used its also ok someone suggested me for better